.changes/next-release/bugfix-AWSSDKforJavav2-466bf4c.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "Fixed the bug \"ArrayIndexOutOfBoundsException\" when single \"=\" as the query string"
```

Comment:
Can we be specific about the  fix is made in SdkHttpFullRequest 
```
Fixed an issue in SdkHttpFullRequest where constructing with a query string consisting of a single \"=\" would throw an ArrayIndexOutOfBoundsException..
```

or something that is more specific ?